packages/eslint-plugin/src/rules/no-unsafe-enum-comparison.ts

```diff
@@ -66,6 +66,19 @@ export default util.createRule({
       'BinaryExpression[operator=/=|<|>/]'(
         node: TSESTree.BinaryExpression,
       ): void {
+        // Exempt bit shift operators since they are not comparisons
+        //
+        // ```ts
+        // enum Fruit {
+        //  Apple,
+        // }
+        //
+        // const myBitShift = 1 << Fruit.Apple;
+        // ```
+        if (node.operator === '>>' || node.operator === '<<') {
```

Comment:
[Suggestion] We already have some exclusions in the in the `'BinaryExpression[operator=/=|<|>/]'` [esquery](https://github.com/estools/esquery). How about adding a `!=` to exclude these? `[operator=/.../][operator!=/<<|>>/]`?

There's definitely a way to get this working in just one regexp if you really want. 😄

Reply:
Please see [3d812c6](https://github.com/typescript-eslint/typescript-eslint/pull/7074/commits/3d812c69909794f8ede28c717cf86fbae5cabf63).